### DISCUSSION TITLE
student developer pack

### DISCUSSION THREAD

**Discussion by vikash000x** [community/community#110302] [GitHub Education]
### Select Topic Area

General

### Body

my student developer pack application has been approved 2 days ago,  but I am not able to access it , that's why I am also not able to fill up application for github campus expert program.  somebody plss help me .. last date is 29th feb for application subbmission.

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @vikashsinha2603 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Reply by vikash000x** [reply]
thankss sir............ pls  help me to get rid out of this problem
